pkg/safe/state.go

```diff
@@ -299,13 +299,20 @@ func (l *List[T]) Find(fn func(T) bool) (T, bool) {
 	return zero, false
 }
 
+// Iterator returns a new iterator over the list.
+func (l *List[T]) Iterator() ListIterator[T] {
```

Comment:
any particular reason to deprecate, or just for easier use?

Reply:
Just easier to use. I don't remember why I didn't do this in the first place. It looks better too!

Reply:
yes, feels better than having to call safe.Iterator